Sources/GoogleAI/ModelContent.swift

```diff
@@ -79,10 +86,12 @@ public struct ModelContent: Codable, Equatable {
         let mimetype = try dataContainer.decode(String.self, forKey: .mimeType)
         let bytes = try dataContainer.decode(Data.self, forKey: .bytes)
         self = .data(mimetype: mimetype, bytes)
+      } else if let functionCall = try? values.decode(FunctionCall.self, forKey: .functionCall) {
```

Comment:
Why use optional `try?` here instead of regular `try`?

Reply:
This was done to be consistent with the `.text` decoding approach but after our discussion I think it's better to check `values.contains(.functionCall)` first and then decode with `try`. This way we'll see the `DecodingError` for debugging help if decoding ever fails when there's a `function_call` field in the payload.

Thanks for the suggestion! Will send out another PR to make this consistent for `.text` and `.data` in `main`.